spec/system/homepage_spec.rb

```diff
@@ -8,12 +8,12 @@
   let!(:organization) do
     create(
       :organization,
-      name: "Decidim Application",
```

Comment:
I would keep this test as is, without any modification, unless we have customizations, which is not the case